src/config.ts

```diff
@@ -62,6 +63,8 @@ enum EnvVarType {
     JsonStringList,
 }
 
+const JSON_CONFIGS_DIR = '/app/jsonConfigs';
```

Comment:
I would make `JSON_CONFIGS_DIR` configurable from the deployment side. E.g. read the `JSON_CONFIGS_DIR` from an environment variable (like `JSON_CONFIGS_DIR`) and fall back to `/app/jsonConfigs` if the envVar is not set. This will allow more flexibility for the Deployment configuration: we can choose the path we want to mount all the config maps and then provide the path to the config dir via `JSON_CONFIGS_DIR` envVar.

Reply:
Oh nice, I like that a lot